js/src/elements/form-behavior.ts

```diff
@@ -356,10 +358,27 @@ export class FormBehaviorElement extends HTMLElement {
 
 	validateRequired(input: HTMLValueElement, label: string): string | null {
 		if (input.validity.valueMissing) {
-			return `${this.locStrings.inputRequired.replace(
+			return this.locStrings.inputRequired.replace(
 				'{inputLabel}',
 				customElements.get(input.localName) ? `A selection for "${label}"` : label
-			)}`;
+			);
+		}
+		return null;
+	}
+
+	validateGroupRequired(input: HTMLValueElement, label: string): string | null {
+		if (input.type === 'radio' || input.type === 'checkbox') {
+			const group = getField(input);
+			const inputs = Array.from(group.querySelectorAll<HTMLInputElement>('input'));
+			// Return null if a single checkbox, so that validateRequired can handle it
+			if (inputs.length === 1 && inputs[0].type === 'checkbox') {
```

Comment:
I'm curious about the case where there is more than 1 checkbox, but neither are required? Can we still submit?

Reply:
Whole method ended up being superfluous.  Removed.